### ISSUE TITLE
On .devcontainer, after running c2g on DGI, we end with an error after the graph is built

### ISSUE THREAD

**Issue by rahlk** [konveyor/tackle-data-gravity-insights#29]
**Describe the bug**
On `.devcontainer`, after running c2g on DGI, we end with an error __after__ the graph is built

**To Reproduce**
1. Spin up `.devcontainer` on vscode
2. Execute `dgi --abstraction=class --quiet c2g --input=tests/fixtures/doop_out`
3. We get the following dump:

**Expected behavior**
We should end with the statement `code2graph build complete`

**Screenshots**
```bash
➜  /app git:(main) ✗ dgi --abstraction=class --quiet c2g --input=tests/fixtures/doop_out
code2graph generator started...
Building Graph...
100%|████████████████████████████████████████████| 1/1 [00:00<00:00, 38.51it/s]
100%|████████████████████████████████████████████| 4/4 [00:00<00:00, 47.19it/s]
100%|██████████████████████████████████████████| 11/11 [00:00<00:00, 32.78it/s]
code2graph build complete
Exception ignored in: <function Driver.__del__ at 0x7f5207778ee0>
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/neo4j/__init__.py", line 279, in __del__
  File "/usr/local/lib/python3.9/site-packages/neo4j/__init__.py", line 309, in close
  File "/usr/local/lib/python3.9/site-packages/neo4j/io/__init__.py", line 787, in close
  File "/usr/local/lib/python3.9/site-packages/neo4j/io/__init__.py", line 776, in remove
  File "/usr/local/lib/python3.9/site-packages/neo4j/io/_bolt4.py", line 295, in close
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1434, in debug
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1589, in _log
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1599, in handle
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1661, in callHandlers
  File "/usr/local/lib/python3.9/logging/__init__.py", line 952, in handle
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1186, in emit
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1175, in _open
NameError: name 'open' is not defined
Exception ignored in: <function Bolt.__del__ at 0x7f51ef13c8b0>
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/neo4j/io/__init__.py", line 379, in __del__
  File "/usr/local/lib/python3.9/site-packages/neo4j/io/_bolt4.py", line 295, in close
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1434, in debug
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1589, in _log
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1599, in handle
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1661, in callHandlers
  File "/usr/local/lib/python3.9/logging/__init__.py", line 952, in handle
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1186, in emit
  File "/usr/local/lib/python3.9/logging/__init__.py", line 1175, in _open
NameError: name 'open' is not defined
➜  /app git:(main) ✗ 
```

**Comment by rofrano** [comment]
@rahlk I fixed this in my branch. It's actually a bug in the new version of `simple-ddl-parser`. I locked it down `0.25.0`

To fix it just change the `setup.py` to use `==` instead if `>=`:
```
simple-ddl-parser==0.25.0
```

**Comment by rofrano** [comment]
This was fixed with #26 KeyError thrown when NEO4J_BOLT_URL not set
